tests/test_xbee_block.py

```diff
@@ -0,0 +1,38 @@
+from ..xbee_block import XBee
+from time import sleep
+from collections import defaultdict
+from unittest import skipIf
+from unittest.mock import MagicMock, patch
+from nio.util.support.block_test_case import NIOBlockTestCase
+
+
+xbee_available = True
+try:
+    import xbee
+except:
+    xbee_available = False
+
+
+@skipIf(not xbee_available, 'xbee is not available!!')
```

Comment:
For future reference: [`skipUnless()`](https://docs.python.org/3/library/unittest.html#unittest.skipUnless)